operation/fhir-operation-validate/src/main/java/com/ibm/fhir/operation/validate/ValidateOperation.java

```diff
@@ -46,14 +47,26 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
             String versionId, Parameters parameters, FHIRResourceHelpers resourceHelper, SearchHelper searchHelper) throws FHIROperationException {
         try {
             Parameter resourceParameter = getParameter(parameters, "resource");
-            if (resourceParameter == null) {
+            String resourceTypeName = resourceType.getSimpleName();
+            Resource resource;
+            // @implNote if $validate is on resource-instance level, fetch the resource from database.
```

Comment:
no need to use "@implNote" here...thats more for use in javadocs 
it allows you to add some extra context to the javadoc without overriding the parent's javadoc.

Reply:
```suggestion
            // if $validate is on resource-instance level, fetch the resource from database.
```

Reply:
this has been fixed. removed "@implNote"